code/game/gamemodes/colonialmarines/main.dm

```diff
@@ -144,7 +149,7 @@
 	. = 1
 	for(var/i in humans)
 		var/mob/living/carbon/human/H = i
-		if(H && H.mind && H.client)
+		if(H && H.mind && H.client && !locate(/obj/item/organ/body_egg/alien_embryo) in H.internal_organs)
```

Comment:
isn't there any defined proc to shorthand the alien embryo check?if not it should be made so that if we change the path we won't have to fix 1300 files